### PR TITLE
Make sure that shared drush command folder is present

### PR DESCRIPTION
**Pull request by patcon**
What about including this in init.pp, like @tizzo has done at @zivtech:
https://github.com/zivtech/vagrant-development-vm/blob/master/puppet-modules/drush/manifests/init.pp#L29-L42

``` puppet
  file { "/usr/share/drush":
    ensure => directory,
    owner => root,
    group => root,
    mode => 755,
  }

  file {  "/usr/share/drush/commands":
    require => File['/usr/share/drush'],
    ensure => directory,
    owner => root,
    group => root,
    mode => 755,
  }
```

Still new to puppet, but I think the rationale is so that globally-installed drush commands can depend on the drush module without redeclaring:
https://github.com/zivtech/vagrant-development-vm/blob/master/puppet-modules/drushfetcher/manifests/init.pp#L1-L4

### CODE REVIEW

**Comment by patcon** [comment]
Added a version constraint until librarian-puppet fixes up its expectation of strict syntax

See: https://github.com/rodjek/librarian-puppet/pull/94

**Comment by erikwebb** [comment]
Because I am just relying on Pear to install Drush, couldn't it be located in different locations based on per-user configuration? I've personally installed Drush through Pear in my home directory on shared servers because of permissions. I know that specifically will not be a problem using Puppet, but it is a hard requirement that I think might not be appropriate. In addition, this should be generally handled by making sure it is installed via pear::package.

To put it another way, if installing via RPM, do you ensure those directories explicitly exist?

**Comment by patcon** [comment]
You raise a good point. What if we just made the dir configurable but defaulting to global (since its the pear default)? Would it be reasonable to expect that at least one command directory is going under drush's control? That way, it could always be inherited like drushfetcher does above, to ensure that it's possible for other modules to install drush commands.

@tizzo thoughts?
